### ISSUE TITLE
Embbeded script editor

### ISSUE THREAD

**Issue by Thelonedevil** [OTBProject/OTBWebInterface#5]
Add an embbeded script editor - see [ace](http://ace.c9.io/#nav=about)

**Comment by NthPortal** [comment]
I feel like this is slightly lower priority than other web interface features (e.g. config editing), because most people are not technically capable of modifying scripts.

**Comment by Thelonedevil** [comment]
I agree, although having a json editor for an "advanced" form of the config editor may be another appliction of this
